app/graph/types/project_media_type.rb

```diff
@@ -85,9 +85,14 @@
 
   field :annotations_count do
     type types.Int
+    argument :annotation_type, types.String
 
-    resolve ->(project_media, _args, _ctx) {
-      project_media.get_versions_log_count
+    resolve ->(project_media, args, _ctx) {
```

Comment:
The changes on `annotations_count` was added before the need to get multiple annotation types and I forgot to include this case. I'll include it now.
About the `blank` case, it was already there and probably is being used somewhere. We can create the `log` and find who uses it and rename there